src/guessTheNumber.js

```diff
@@ -0,0 +1,18 @@
+const min = Math.ceil(1)
+const max = Math.floor(100)
+const randomNum = Math.floor(Math.random() * (max - min + 1) + min)
+const getUserInput = document.querySelector('input#guess-number')
+const guessTheNumber = (inputValue) => {
```

Comment:
You can comma separate variable declarations and use const only once. All the cool kids are doing it:
```
const min = Math.ceil(1),
  max = Math.floor(100),
  randomNum = Math.floor(Math.random() * (max - min + 1) + min), 
  getUserInput = document.querySelector('input#guess-number'),
  guessTheNumber = (inputValue)...
```